packages/main/src/themes/sap_fiori_3_dark/SliderBase-parameters.css

```diff
@@ -0,0 +1,5 @@
+@import "../base/SliderBase-parameters.css";
+
+:root {
+	--_ui5_slider_inner_min_width: 4rem;

```

Comment:
Does not this  belong to the base param file - the min-width should be 4rem across all themes right?

Reply:
In our visual spec. it is only specified for the Fiori 3 theme, Belize doesn't have defined min-width for the Sliders